bayesian_cnn_prometheus/learning/bayesian_detector.py

```diff
@@ -57,7 +57,8 @@ def _initialize_model(self, input_shape: Tuple[int, ...]):
         self._model = BayesianVnet(input_shape, kernel_size=self._kernel_size, activation=self._activation,
                                     padding=self._padding, prior_std=self._prior_std)
         self._model.summary(line_length=127)
-        loss_function = variational_free_energy_loss(self._kl_alpha)
+        self._model(tf.ones((self._batch_size, *input_shape)))
+        loss_function = variational_free_energy_loss(self._model, self._kl_alpha)
```

Comment:
Tak było przy tf1 kiedy używaliśmy customowej funkcji straty, więc przywróciłem. 
Nie whodziłem w to czy to jest konieczne